int16[] blocks_over
int16 gripper_over
bool gripper_closed

